ts/store/reducers/profile.ts

```diff
@@ -81,7 +81,7 @@ const reducer = (
       }
 
     case getType(profileUpsert.failure):
-      return pot.toError(state, action.payload);
+      return pot.toError(state, Error(action.payload));
```

Comment:
for consistency you may want to change `ProfileState` to `pot.Pot<UserProfileUnion, string>;`